resources/sample-content.edn

```diff
@@ -0,0 +1,4236 @@
+{:metabase_database
+ ({:description "Some example data for you to play around with.",
+   :cache_field_values_schedule "0 0 17 * * ? *",
+   :timezone "UTC",
+   :auto_run_queries true,
+   :metadata_sync_schedule "0 26 * * * ? *",
+   :name "Sample Database",
+   :settings nil,
+   :caveats
+   "You probably don't want to use this for your business-critical analyses, but hey, it's your world, we're just living in it.",
+   :creator_id nil,
+   :is_full_sync true,
+   :updated_at #t "2024-04-01T15:03:43.314320Z",
+   :cache_ttl nil,
+   :details
+   "{\"db\":\"file:/Users/callumherries/meta/metabase/plugins/sample-database.db;USER=GUEST;PASSWORD=guest\"}",
```

Comment:
Should this be here?

Reply:
Ha. thanks for the reminder, I'll delete this in the final version :D The `details` get updated later to reflect the real location of the sample DB, but I'll delete this manually anyway. gotta keep the JAR size down